<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>501</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit_2">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_NewFile"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_AllSelect"/>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="action_zhushi"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_Description"/>
   </widget>
   <widget class="QMenu" name="menu_D">
    <property name="title">
     <string>调试(&amp;D)</string>
    </property>
    <addaction name="action_Run"/>
    <addaction name="action_Run_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_D"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_NewFile"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="action_Undo"/>
   <addaction name="action_Redo"/>
   <addaction name="action_Run"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_NewFile">
   <property name="icon">
    <iconset theme="file" resource="myresources.qrc">
     <normaloff>:/myImages/image/create.png</normaloff>:/myImages/image/create.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myImages/image/open.png</normaloff>:/myImages/image/open.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myImages/image/save.png</normaloff>:/myImages/image/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myImages/image/reback.png</normaloff>:/myImages/image/reback.png</iconset>
   </property>
   <property name="text">
    <string>重做(&amp;Y)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myImages/image/about.png</normaloff>:/myImages/image/about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_Description">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myImages/image/Description.png</normaloff>:/myImages/image/Description.png</iconset>
   </property>
   <property name="text">
    <string>说明文档</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myImages/image/back.png</normaloff>:/myImages/image/back.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_AllSelect">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myImages/image/select_all.png</normaloff>:/myImages/image/select_all.png</iconset>
   </property>
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myImages/image/cut.png</normaloff>:/myImages/image/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myImages/image/copy.png</normaloff>:/myImages/image/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myImages/image/paste.png</normaloff>:/myImages/image/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Run">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myImages/image/run.png</normaloff>:/myImages/image/run.png</iconset>
   </property>
   <property name="text">
    <string>编译</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myImages/image/saveas.png</normaloff>:/myImages/image/saveas.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_Run_2">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/myImages/image/run.png</normaloff>:/myImages/image/run.png</iconset>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="action_zhushi">
   <property name="text">
    <string>注释</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+/</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections/>
</ui>
